EliasHjelm/mumsig
# Feature/calculate ingredients portions

- [x] First install npm-package with npm install

Every recipe card on the start page has a link to respective recipe-page. 
Dropdown with portions and ingredients list are located on the right. Ingredients' quantities are re-calculated by selected option from portions' dropdown and depends on default portion's value.

Test cases:

1. click on recipe card on the start page.

Result: 
- location path changed /recipes/:_id, window navigated to recipe page
- Ingredients are listed, amount appeared

2. Click on dropdown with portions and select another option

Result:
- ingredients are re-calculated

3. Special case - click on "Snabba vanilj- och bärbullar" on the start page or navigate to http://localhost:3000/recipes/8. Check ingredients section.

Result: 
- ingredients are divided into groups ("Deg", "Fyllning", "Kristyr", "Dekorering"), every group's name has semi-bold font-wight.

Notes:

1. Ingredients amount appeared as fraction (e.g. 1/4 dl). In certain cases it looks weird a little (5/6 citron), fix is coming next sprint
2. Dividing into groups required a modification of Recipe model
3. Almost all ingredients are copied from src\components\CreateRecipe\food-data.json with original names. They look strange, need some fix with e.g. free text field in Create recipe form
